README.md

```diff
@@ -1,2 +1,56 @@
+[![GitHub Issues or Pull Requests](https://img.shields.io/github/issues-pr/team-zomsa/aplib.net)](https://github.com/team-zomsa/aplib.net/pulls)
+[![GitHub deployments](https://img.shields.io/github/deployments/team-zomsa/aplib.net/github-pages?label=doxygen)
+](https://team-zomsa.github.io/aplib.net/)
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=team-zomsa_aplib.net&metric=alert_status)](https://sonarcloud.io/project/overview?id=team-zomsa_aplib.net)
+[![SonarCloud Coverage](https://sonarcloud.io/api/project_badges/measure?project=team-zomsa_aplib.net&metric=coverage)](https://sonarcloud.io/project/overview?id=team-zomsa_aplib.net)
+[![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=team-zomsa_aplib.net&metric=duplicated_lines_density)](https://sonarcloud.io/project/overview?id=team-zomsa_aplib.net)
+[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=team-zomsa_aplib.net&metric=code_smells)](https://sonarcloud.io/project/overview?id=team-zomsa_aplib.net)
+[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=team-zomsa_aplib.net&metric=reliability_rating)](https://sonarcloud.io/project/overview?id=team-zomsa_aplib.net)
+[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=team-zomsa_aplib.net&metric=security_rating)](https://sonarcloud.io/project/overview?id=team-zomsa_aplib.net)
+[![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=team-zomsa_aplib.net&metric=sqale_index)](https://sonarcloud.io/project/overview?id=team-zomsa_aplib.net)
+[![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=team-zomsa_aplib.net&metric=vulnerabilities)](https://sonarcloud.io/project/overview?id=team-zomsa_aplib.net)
+
+
+## Overview
+Aplib.Net is a C# re-implementation of the Agent Programming LIBrary (Aplib) for Java.
+The original Aplib can be found [here](https://github.com/iv4xr-project/aplib) and is licensed to Utrecht University, created under the [iv4XR project](https://iv4xr-project.eu/).
+
+Aplib.Net provides a set of tools for creating automatic playtesting agents for games. It is designed in such a way that it can be used in any game that uses the C# language. However, we provide additional support for the [Unity game engine](https://unity.com/).
+
+To show the functionality of the library, a demo game has also been created and can be found [its respective repository](https://github.com/team-zomsa/aplib.net-demo).
+
+
+## Installation
+Aplib.Net is available as a NuGet package or as a simple dll file, that can then be added as a reference.
+
+Both are available in the [GitHub Release](https://github.com/team-zomsa/aplib.net/releases).
+
+An easier way of installing is also provided using the NuGet Gallery at NuGet.org, which can be found [here](https://www.nuget.org/packages/Aplib.Core/).
+
```

Comment:
Perhaps say a thing or two about installing it in Unity, as it is not straightforward.